iuf

```diff
@@ -144,6 +151,21 @@ def validate_stages(config):
             errors.append(f"{param_name} {arg_value} was specified, but the file could not be found")
             return
         arg_val_basename = os.path.basename(arg_value)
+        if not os.path.exists(media_dir):
+            if in_bootprep_stage:
+                # The media directory is necessary during the bootprep stage
+                # because bootprep files are copied to the media directory.
```

Comment:
Can someone educate me on why this is the case?

Reply:
Because the bootprep files need to be in the media directory.

Reply:
I should mention: I think it would be unusual to hit this error.  By the time we're running the bootprep stages, we should  have a media directory, unless it's been blown away.

Reply:
might it be worth storing media-dir in activity/stages so if someone were to re-run just the bootprep stage they wouldn't need to re-specify the media-dir?

Reply:
@larry-tordsen so this means the user has to specify ```-m``` for process-media, update-cfs-config, and prepare-images....do I have that right?  Just want to make sure the docs are accurate.

Reply:
Yes.  *But* -- if they use the default activity name for the media directory.  If the activity is myactivity.5, and they put their stuff at `/etc/cray/upgrade/csm/myactivity.5`, they shouldn't have to specify the media directory at all.